From 7ea3edf07a30e524e300dd99e1d8cef040beac55 Mon Sep 17 00:00:00 2001
From: prog <youssef@airspy.com>
Date: Tue, 17 Apr 2018 15:23:23 +0200
Subject: [PATCH 2/7] Disabled the test code.

---
 libairspyhf/src/airspyhf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libairspyhf/src/airspyhf.c b/libairspyhf/src/airspyhf.c
index 626bfdf..1351bf6 100644
--- a/libairspyhf/src/airspyhf.c
+++ b/libairspyhf/src/airspyhf.c
@@ -1381,4 +1381,4 @@ int ADDCALL airspyhf_set_lib_dsp(airspyhf_device_t* device, const uint8_t flag)
 	return AIRSPYHF_SUCCESS;
 }
 
-#include "special.c"
+//#include "special.c"
-- 
2.11.0

